packages/vue/src/auth/service.ts

```diff
@@ -152,4 +152,20 @@ export class FronteggAuthService implements FronteggPluginService {
       }
     }
   };
+
+
+  getCurrentRoute = () => {
+    let route: string = '/';
+    if (this.router?.currentRoute?.path) {
+      // vue-router v3
+      route = this.router?.currentRoute?.path;
+    }
+    // @ts-ignore vue-router v4
+    if (this.router?.currentRoute?.value?.path) {
+      // @ts-ignore vue-router v4
+      route = this.router?.currentRoute?.value?.fullPath;
+    }
+
+    return route
+  };
```

Comment:
maybe we can write typeguard for this that will indicate if is vue 3 or 4 router?

Reply:
we cannot add types for this one, I fixed the fullPath at line 164, we need to replace it with router proxy to support multiple versions.